<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FUst</class>
 <widget class="QWidget" name="FUst">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>938</width>
    <height>902</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Установки для обработки информации</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/res/Equipment.png</normaloff>:/res/Equipment.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Название АРМ:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_nazkomp">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Ресурс для размещения необработанных &quot;сырых&quot; измерений:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="le_si"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="btnSir">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="res.qrc">
            <normaloff>:/res/1/folder_edit.png</normaloff>:/res/1/folder_edit.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Папка для размещения временных файлов:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="le_temp"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="btnTemp">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="res.qrc">
            <normaloff>:/res/1/folder_edit.png</normaloff>:/res/1/folder_edit.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Интервал запуска пакета обработки (сек.):</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="le_km"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QCheckBox" name="cb_pravt">
          <property name="text">
           <string>Признак автоматической обработки</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
     <zorder></zorder>
     <zorder></zorder>
     <zorder></zorder>
     <zorder></zorder>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QFrame" name="frame_2">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_14">
      <item>
       <widget class="QFrame" name="frame_6">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QCheckBox" name="cb_obrsi">
             <property name="text">
              <string>Помещать необработанную &quot;сырую&quot; информацию в БД</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="le_obrsi"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_7">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_12">
           <item>
            <widget class="QCheckBox" name="cb_obr11_15">
             <property name="text">
              <string>Конвертировать файл ППСИ (ДДС КВИ, ДДС КВНО)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="le_obr11_15"/>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_13">
           <item>
            <widget class="QCheckBox" name="cb_obr12">
             <property name="text">
              <string>Конвертировать файл ППСИ (ВРВ)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="le_obr12"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>407</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QFrame" name="frame_3">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_19">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Обработка ДДС</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QCheckBox" name="cb_obr14">
             <property name="text">
              <string>Разделить файл КВИ по суткам</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="le_obr14"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_15">
           <item>
            <widget class="QCheckBox" name="cb_obr13">
             <property name="text">
              <string>Объединить файлы КВИ в суточные</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="le_obr13"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_16">
           <item>
            <widget class="QCheckBox" name="cb_obr17">
             <property name="text">
              <string>Разделить файл КВНО по суткам</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="le_obr17"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_17">
           <item>
            <widget class="QCheckBox" name="cb_obr16">
             <property name="text">
              <string>Объединить файлы КВНО в суточные</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="le_obr16"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Обработка ВРВ</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QCheckBox" name="cb_obr19">
             <property name="text">
              <string>Разделить файл ALTI по суткам</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="le_obr19"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_18">
           <item>
            <widget class="QCheckBox" name="cb_obr18">
             <property name="text">
              <string>Объединить файлы ALTI в суточные</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="le_obr18"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>571</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QFrame" name="frame_4">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_27">
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>Обработка БДС</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_20">
           <item>
            <widget class="QCheckBox" name="cb_obr2">
             <property name="text">
              <string>Проверить и переименовать измерительный файл</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="le_obr2"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_21">
           <item>
            <widget class="QCheckBox" name="cb_obr3">
             <property name="text">
              <string>Разделить измерительный файл по суткам</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="le_obr3"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_22">
           <item>
            <widget class="QCheckBox" name="cb_obr4">
             <property name="text">
              <string>Объединить измерительные файлы в суточные</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="le_obr4"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_23">
           <item>
            <widget class="QCheckBox" name="cb_obr7">
             <property name="text">
              <string>Разделить навигационный файл по принадлежности к ГНСС</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="le_obr7"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_24">
           <item>
            <widget class="QCheckBox" name="cb_obr8">
             <property name="text">
              <string>Разделить навигационный файл по суткам</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="le_obr8"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_25">
           <item>
            <widget class="QCheckBox" name="cb_obr9">
             <property name="text">
              <string>Объединить навигационные файлы в суточные для всех НПН</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="le_obr9"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_26">
           <item>
            <widget class="QCheckBox" name="cb_obr10">
             <property name="text">
              <string>Конвертировать L-файл в формат RINEX 3.02</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="le_obr10"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cb_obrzapr">
        <property name="text">
         <string>Обработка автоматических запросов</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>165</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
     <zorder>groupBox_3</zorder>
     <zorder>horizontalSpacer_3</zorder>
     <zorder>cb_obrzapr</zorder>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QFrame" name="frame_5">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QPushButton" name="btnOk">
        <property name="text">
         <string>Сохранить</string>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/res/OK.png</normaloff>:/res/OK.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnCancel">
        <property name="text">
         <string>Отменить</string>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/res/No.png</normaloff>:/res/No.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSave">
        <property name="text">
         <string>Сохранить</string>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/res/1/table_save.png</normaloff>:/res/1/table_save.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnLoad">
        <property name="text">
         <string>Загрузить</string>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/res/1/table_import.png</normaloff>:/res/1/table_import.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>844</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
